src/components/MovieItem/eventHandler.ts

```diff
@@ -0,0 +1,64 @@
+import isElement from '../../utils/isElement';
+import isHTMLElement from '../../utils/isHTMLElement';
+import Modal from '../Modal/Modal';
+import { openModalHandler } from '../Modal/eventHandler';
+import { MovieDetailProps } from '../../types/movie';
+import MovieStorageService from '../../services/MovieStorageService';
+
+/* eslint-disable max-lines-per-function */
+const renderMovieItemDetailModal = ({
+  title,
+  genres,
+  vote_average,
+  poster_path,
+  overview,
+  star_rating,
+}: MovieDetailProps) => {
+  Modal({
+    title,
+    genres,
+    vote_average,
+    poster_path,
+    overview,
+    star_rating,
+  });
+  openModalHandler();
+};
+
+const renderDetailModal = (movieDetailData: MovieDetailProps) => {
+  const { title, genres, vote_average, poster_path, overview, star_rating } = movieDetailData;
+  renderMovieItemDetailModal({
+    title,
+    genres,
+    vote_average,
+    poster_path,
+    overview,
+    star_rating,
+  });
+};
+
+const getMovieTitleEventPhaseHandler = async (event: Event, ul: Element) => {
+  if (!(event.target instanceof Element)) return;
+  const li = event.target.closest('li');
```

Comment:
```suggestion
  if (!(event.target instanceof Element)) return;
  
  const li = event.target.closest('li');
```

1. 여기서 개행을 주는 것이 낫지 않을까요?
2. 그런데 event.target Element 객체를 받지 못하면 return해서 핸들러 함수를 실행하지 않으면 될까요?
   - 이 함수를 사용하는 사람은 개발자일 것 같은데 오류를 터뜨리는 것이 더 낫지 않을까 싶습니다. (제 생각)
      - 개발자가 event.target 객체가 Element가 아니어서 return된 경우를 빠르게 알아차릴 수 있을까요?

Reply:
1. 이번에 미션 기준을 10줄로 잡고 함수를 구현하다보니까, 개행 부분을 일부러 줄인 부분들이 많이 있습니다..! 이미 구현을 끝낸 함수가 11-12줄인경우 개행만 줄여주면 된다는 생각에 그렇게 된것 같아요.

2. 
맞는 말씀이에요! 사실 요소를 찾고나서 해당 요소가 없으면 예외처리를 해주는게 더 맞는 동작이라고 생각합니다. 함수가 10줄이라는 생각에 예외처리 또한 저 함수 안에 들어가야한다는 생각에 기반해서 놓친거 같습니다. 짚어주셔서 감사해요! 한번 더 고심해보도록 하겠습니다~!  🙇